data/campaigns/tutorial/achievements.cfg

```diff
@@ -2,5 +2,12 @@
 [achievement_group]
     display_name=_"Tutorial"
     content_for=tutorial
-    {ACHIEVEMENT "completed" _"Complete the Tutorial" _"Complete all scenarios of the Tutorial campaign."}
+    [achievement]
+        id="completed"
+        name=_"Complete the Tutorial"
```

Comment:
Wouldn't block this PR about it, but the UI when the player gets this achievement feels wrong. They've played through two scenarios of being told what to do, and at the end a dialog puts up with "Complete the Tutorial" in the imperative. Either that dialog should have an "Achievement completed" header on it, or this text should be past tense such as "Tutorial completed".

Reply:
Updated.